src/main/java/org/jenkinsci/plugins/configfiles/maven/job/MvnGlobalSettingsProvider.java

```diff
@@ -131,13 +134,20 @@ public String getDisplayName() {
             return "provided global settings.xml";
         }
 
-        public ListBoxModel doFillSettingsConfigIdItems(@AncestorInPath ItemGroup context, @AncestorInPath Item project) {
-            project.checkPermission(Item.CONFIGURE);
-            
+        public ListBoxModel doFillSettingsConfigIdItems(@AncestorInPath ItemGroup context, @AncestorInPath Item project, String currentValue) {
+            Permission permToCheck = project == null ? Jenkins.ADMINISTER : Item.EXTENDED_READ;
+            AccessControlled contextToCheck = project == null ? Jenkins.get() : project;
+
             ListBoxModel items = new ListBoxModel();
             items.add("please select", "");
```

Comment:
```suggestion
            items.add(Messages.MvnSettingsProvider_PleaseSelect(), "");
```

Reply:
I'm keeping here the same text as before. Didn't want to use a new one

Reply:
Doesn't hurt to improve, even more if the i18n entry exists.